core/src/zxing/DecodeHints.cpp

```diff
@@ -29,7 +29,7 @@ using zxing::DecodeHints;
 // VC++
 using zxing::BarcodeFormat;
 
-const DecodeHintType DecodeHints::CHARACTER_SET;
```

Comment:
This is the only change we need, just this one line to be deleted, that should be the entire pull request. Also so should just be one commit. Please just force push the change (so you don't need to open a new pull request).